src/Badge/Model/Package.php

```diff
@@ -107,7 +107,7 @@ private function calculateLatestVersions(): void
     private function getBranchAliases(Version $version): ?array
     {
         $extra = $version->getExtra();
-        if (null !== $extra && isset($extra['branch-alias']) && \is_array($extra['branch-alias'])) {
+        if (isset($extra['branch-alias']) && \is_array($extra['branch-alias'])) {
```

Comment:
In the new version is it impossible that it is null?

Reply:
phpStorm was highlighting this as a problem since `$version->getExtra()` method doesn't expect to return a `null` value